pkgs/development/python-modules/protobuf/default.nix

```diff
@@ -104,6 +104,8 @@ buildPythonPackage {
     #   https://github.com/protocolbuffers/protobuf/commit/5abab0f47e81ac085f0b2d17ec3b3a3b252a11f1
     #
     "google/protobuf/internal/generator_test.py"
+  ] ++ lib.optionals (lib.versionAtLeast protobuf.version "25") [
+    "minimal_test.py" # ModuleNotFoundError: No module named 'google3'
```

Comment:
Is it the interpretation that `google3` is a package that comes with upb?

Reply:
I don't think so. Grepping through the repo just makes it look like an oversight during moving code around.

Reply:
Let's update the comment to reflect that, "Refers to a non-existent module 'google3'. Presumably an oversight". An issue upstream?

Reply:
I mentioned it in the commit message. My rationale was that I wanted to keep the comment short as people can always check if they get the same error message.